requirements.txt

```diff
@@ -1,14 +1,14 @@
-Django==1.4.18
+Django==1.7.7
 Markdown==2.2.1
 MySQL-python==1.2.5
 PyYAML==3.10
 Pygments==1.5
-django-allauth==0.9.0
-django-bootstrap-form==2.0.3
-django-filter==0.5.4
-django-tastypie==0.9.15
-django-uni-form==0.7.0
-djangorestframework==2.1.17
+django-allauth==0.19.1
+django-bootstrap-form==3.2
+django-filter==0.9.2
+django-tastypie==0.12.1
+django-uni-form==0.9.0
+djangorestframework==3.1.1
```

Comment:
while we're at it, is `django-bootstrap-form`, `django-filter`, `django-tastypie`, or `djangorestframework` even used?  I also find it very strange that we have both of the two major competing API libraries (tastypie and DRF)....sigh